### PR TITLE
fix: don't join absolute paths with baseUrl

### PR DESCRIPTION
**Pull request by npdev453**
fixes:
#181

### CODE REVIEW

File: src/mapping-entry.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ export function getAbsoluteMappingEntries(
   for (const key of sortedKeys) {
     absolutePaths.push({
       pattern: key,
-      paths: paths[key].map((pathToResolve) =>
+      paths: paths[key].map((pathToResolve) =>                    
+        if (path.isAbsolute(pathToResolve)) return pathToResolve;
         path.join(absoluteBaseUrl, pathToResolve)
```
Comment (rattrayalex) [inline src/mapping-entry.ts:36]: It looks like this is missing curly braces; you probably want this:
```ts
      paths: paths[key].map((pathToResolve) => {                 
        if (path.isAbsolute(pathToResolve)) return pathToResolve;
        return path.join(absoluteBaseUrl, pathToResolve);
     },

Comment (npdev453) [reply src/mapping-entry.ts]: fixed

Comment (rattrayalex) [reply src/mapping-entry.ts]: Wow quick fix, thanks! Hope this is merged soon.

Comment (npdev453) [reply src/mapping-entry.ts]: @jonaskello

**Review by aleclarson — CHANGES_REQUESTED** [review]

**Review by aleclarson — APPROVED** [review]

**Comment by jonaskello** [comment]
@npdev453 Could you also add an entry for this PR in the CHANGELOG under unreleased heading ? With that it should be good to merge.

**Comment by npdev453** [comment]
> @npdev453 Could you also add an entry for this PR in the CHANGELOG under unreleased heading ? With that it should be good to merge.

Done.

**Comment by jonaskello** [comment]
@aleclarson I think this PR made some tests fail. Not sure if the tests are invalid or if something needs to be fixed. Could you take a look at it or perhaps we should just revert this PR?

**Pull request merged**